health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/createUpdatePlanProject.js

```
@@ -398,56 +420,61 @@ const createUpdatePlanProject = async (req) => {
+
         const updatedPlanObjFormula = {
           ...fetchedPlanForFormula,
-          operations:[
-            ...prevFormulas,
-            ...formulasToUpdate
-          ]
-        }
+          operations: [...prevFormulas, ...formulasToUpdate],
+        };
 
         const planResFormula = await updatePlan(updatedPlanObjFormula);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)